app/controllers/admin/collections_controller.rb

```diff
@@ -8,14 +9,9 @@ def index
     # We're assuming if you can view the index, you can see all published and
     # unpublished collections.
 
-
-    # NOTE WELL: To use ransack, all attributes we want ransack to search or sort
-    # on NEED TO be listed in Colletion.ransackable_attributes and/or Collection.ransackable_associations
-    @q = Collection.ransack(params[:q]).tap do |ransack|
-      ransack.sorts = 'title asc' if ransack.sorts.empty?
-    end
-
-    scope = @q.result
+    # Searching, filtering, sorting and pagination.
+    params.delete("button")
```

Comment:
I don't think you need to delete this from params for any reason? just an unnecessary line?